tests/Http/GroupTypeTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Tests\Http;
+
+use Tests\TestCase;
+use Rogue\Models\GroupType;
+use Illuminate\Database\QueryException;
+
+class GroupTypeTest extends TestCase
+{
+    /**
+     * Test that a GET request to /api/v3/group-types returns an index of all group types.
+     *
+     * @return void
+     */
+    public function testGroupTypeIndex()
+    {
+        $groupTypes = factory(GroupType::class, 5)->create();
+
+        $response = $this->getJson('api/v3/group-types');
+        $decodedResponse = $response->decodeResponseJson();
+
+        $response->assertStatus(200);
+        $this->assertEquals(5, $decodedResponse['meta']['pagination']['count']);
+
+        foreach ($groupTypes as $groupType) {
+            $this->assertDatabaseHas('group_types', [
+                'name' => $groupType->name,
+            ]);
+        }
```

Comment:
We could probably remove this assertion, since it's effectively just testing that Laravel's model factories work. I think we can generally trust that the framework (and more broadly, anything in the "arrange" portion of the test) works as expected, and limit assertions to whatever application logic is running (the response).